#ifndef _VISUALIZEDETECTEDOBJECTS_HPP
#define _VISUALIZEDETECTEDOBJECTS_HPP

#include <vector>
#include <string>
#include <sstream>
#include <cmath>
#include <iomanip>

#include <ros/ros.h>
#include <std_msgs/Header.h>

#include <visualization_msgs/MarkerArray.h>
#include <visualization_msgs/Marker.h>

#include "uav_msgs/DetectedObject.h"
#include "uav_msgs/DetectedObjectArray.h"





#endif